docs/admin/authorization/index.md

```diff
@@ -136,3 +136,16 @@ As of version 1.3, clusters created by kube-up.sh are configured so that the A
 {% endcapture %}
 
 {% include templates/concept.md %}
+
+## Privilege escalation via pod creation
+
+Users who have ability to create pods in a namespace can potentially escalate their privileges within that namespace.  If a user is granted permission to create pods (or controllers that create pods), the Kubernetes API does not police the privileges of the pods being created.  This means that users can create pods that access secrets the user herself cannot read, or that run under a service account with different/greater permissions.
```

Comment:
"the Kubernetes API does not police the privileges of the pods being created" is a pretty broad statement, and isn't true in some respects:
* the permissions of a pod can be limited by PodSecurityPolicy
* the API calls a pod can make are limited to the API permissions of the pod's service account

It's fine to call out the secret/configmap mounting, and that the ability to access secrets in a namespace (directly or by mounting them into a pod) gives access to service account credentials in that namespace.

Reply:
would be more accurate to say, "Kubernetes access control does not police the contents of the podspec"

I'm trying to give a little context for why it works like this.  If that's not any more accurate, we can just drop that sentence and rework.

Reply:
> would it be more accurate to say, "Kubernetes access control does not police the contents of the podspec"?

Not really... the [PodSecurityPolicy](https://kubernetes.io/docs/concepts/policy/pod-security-policy/) admission plugin lets you control what volumes and privileges a pod spec can contain, and the [ImagePolicyWebhook](https://kubernetes.io/docs/admin/admission-controllers/#imagepolicywebhook) admission plugin lets you control what images your pod can run.

Reply:
It would probably be more helpful to point people to those admission plugins to indicate that the platform has the capability to control pod specs, volumes, and which images are allowed, but unless you take advantage of those controls, anyone creating a pod has access to all secrets/configmaps/serviceaccounts in the namespace.